### PR TITLE
fix(devops): build multi platform images

### PR DESCRIPTION
**Pull request by cnlangzi**
### Changes
- build multi platform docker images for linux/amd64 and linux/arm64

### Tests
Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/blobber/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

### Associated PRs (Link as appropriate):
- 0chain:
- gosdk:
- system_test:
- zboxcli:
- zwalletcli:
- Other: ...

### CODE REVIEW

**Comment by vinayakgautamops** [comment]
@Kishan-Dhakan https://github.com/0chain/blobber/runs/5403337492?check_suite_focus=true

File: docker.local/base.Dockerfile [hunk 1]
```diff
@@ -10,27 +11,29 @@ RUN echo "https://mirrors.aliyun.com/alpine/v3.14/community" >> /etc/apk/reposit
...
-    make -C mcl -j $(nproc) lib/libmclbn256.so install && \
-    cp mcl/lib/libmclbn256.so /usr/local/lib && \
-    make MCL_DIR=$(pwd)/mcl -C bls -j $(nproc) install && \
-    rm -R /tmp/mcl && \
-    rm -R /tmp/bls
+RUN apk add gmp gmp-dev openssl-dev 
+
+WORKDIR /tmp
+RUN wget https://github.com/herumi/mcl/archive/master.tar.gz
+RUN tar zxvf master.tar.gz &&  mv mcl* mcl
```
Comment (Kishan-Dhakan) [inline docker.local/base.Dockerfile]: Would tar -zxvf work?

Comment (vinayakgautamops) [reply docker.local/base.Dockerfile]: yes, it works on my system

Comment (cnlangzi) [reply docker.local/base.Dockerfile]: yes. it works on linux/macos. our github runner is ubuntu. so it is ok.

Comment (cnlangzi) [reply docker.local/base.Dockerfile]: sorry, the shell run on alpine , not ubuntu.

File: docker.local/bin/build.base.sh [hunk 2]
```diff
@@ -3,6 +3,29 @@ set -e
...
+fi
+echo "  DOCKER_BUILD=$DOCKER_BUILD"
+echo "  DOCKER_IMAGE_BASE=$DOCKER_IMAGE_BASE"
 
-docker build --build-arg GIT_COMMIT=$GIT_COMMIT -f docker.local/base.Dockerfile . -t blobber_base
\ No newline at end of file
+echo ""
+echo "2> download herumi"
+
+[ ! -f ./docker.local/bin/mcl-master.tar.gz ] && wget -O ./docker.local/bin/mcl-master.tar.gz https://github.com/herumi/mcl/archive/master.tar.gz
```
Comment (peterlimg) [inline docker.local/bin/build.base.sh]: We'd better get the mcl with specific version so that our building can be deterministic.

**Review by peterlimg — CHANGES_REQUESTED** [review]

File: docker.local/bin/build.base.sh (hunk 2 above)
Comment (cnlangzi) [reply docker.local/bin/build.base.sh]: fixed them with latest tags

**Review by peterlimg — APPROVED** [review]
lgtm

**Pull request merged**